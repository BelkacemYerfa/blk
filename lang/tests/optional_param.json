{
  "ast": [
    {
      "Command": "trim",
      "Order": 3,
      "Body": null,
      "Params": [
        {
          "Col": 6,
          "Kind": "time",
          "Row": 1,
          "Value": "00:00:00"
        },
        {
          "Col": 17,
          "Kind": "time",
          "Row": 1,
          "Value": "00:00:10"
        },
        {
          "Col": 1,
          "Kind": "string",
          "Row": 3,
          "Value": "all"
        }
      ]
    },
    {
      "Command": "trim",
      "Body": null,
      "Order": 7,
      "Params": [
        {
          "Col": 6,
          "Kind": "time",
          "Row": 3,
          "Value": "00:00:00"
        },
        {
          "Col": 17,
          "Kind": "time",
          "Row": 3,
          "Value": "00:00:10"
        },
        {
          "Col": 28,
          "Kind": "string",
          "Row": 3,
          "Value": "out/clip1.mp4"
        }
      ]
    }
  ],
  "tokens": [
    {
      "Col": 1,
      "Kind": "trim",
      "Row": 1,
      "Text": "trim"
    },
    {
      "Col": 6,
      "Kind": "time",
      "Row": 1,
      "Text": "00:00:00"
    },
    {
      "Col": 17,
      "Kind": "time",
      "Row": 1,
      "Text": "00:00:10"
    },
    {
      "Col": 1,
      "Kind": "trim",
      "Row": 3,
      "Text": "trim"
    },
    {
      "Col": 6,
      "Kind": "time",
      "Row": 3,
      "Text": "00:00:00"
    },
    {
      "Col": 17,
      "Kind": "time",
      "Row": 3,
      "Text": "00:00:10"
    },
    {
      "Col": 28,
      "Kind": "string",
      "Row": 3,
      "Text": "out/clip1.mp4"
    },
    {
      "Col": 43,
      "Kind": "end of file",
      "Row": 3,
      "Text": ""
    }
  ]
}
