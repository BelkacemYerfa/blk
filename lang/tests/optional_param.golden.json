{
  "tokens": [
    {
      "type": "comment",
      "value": ""
    },
    {
      "type": "comment",
      "value": ""
    }
  ],
  "ast": [
    {
      "type": "package",
      "name": "tests"
    },
    {
      "type": "import",
      "path": "testing"
    },
    {
      "type": "function",
      "name": "TestPushCommand",
      "params": [],
      "body": []
    }
  ]
}
